client/packages/system/src/Vaccination/api/operations.generated.ts

```diff
@@ -258,17 +250,17 @@ export const VaccinationDetailFragmentDoc = gql`
       lastName
     }
     given
+    givenStoreId
+    item {
+      id
+      name
+    }
```

Comment:
item info should be available on vaccination, even if stock line is not selected/is for different store